app/assets/javascripts/application.js

```diff
@@ -0,0 +1,4 @@
+//= require jquery
+//= require jquery_ujs
+//= require bootstrap-sprockets
+//=require_tree .
```

Comment:
Please remove those unless its necessary.

Reply:
Bhai, these are directive lines, hence these are necessary for the project. Thanks